### ISSUE TITLE
peers problem

### ISSUE THREAD

**Issue by aspirecoin** [ethcore/parity#2314]
```
6-09-25 17:05:35     0/ 0/50 peers     15 MiB db    8 KiB chain    2 KiB queue   11 KiB sync
and 5 hours 0 peers...
 parity --jsonrpc-interface 0.0.0.0 --jsonrpc-hosts all --max-peers 100 --cache-size-db 1024 --author 617200a4b1f67bf0ba4ca722d1f853ad818f8fb5 --geth  --min-peers 50
```
Ubuntu 14.04
port 30303 open 
when i try sync without snapshots peers connect 25-30 peers.. when use snapshots - peers 0
who can help?

**Comment by aspirecoin** [comment]
2016-09-25 17:10:13 Syncing #2021870 7216…5056      0 blk/s    0 tx/s   0 Mgas/s       0+    0 Qed   #2021870    1/ 1/50 peers     15 MiB db   16 KiB chain    2 KiB queue   11 KiB sync
2016-09-25 17:10:44     0/ 1/50 peers     15 MiB db   16 KiB chain    2 KiB queue   11 KiB sync
2016-09-25 17:11:14     0/ 0/50 peers     15 MiB db   16 KiB chain    2 KiB queue   11 KiB sync
2016-09-25 17:11:45     0/ 0/50 peers     15 MiB db   16 KiB chain    2 KiB queue   11 KiB sync
2016-09-25 17:12:15     0/ 0/50 peers     15 MiB db   16 KiB chain    2 KiB queue   11 KiB sync

**Comment by rphmeier** [comment]
This is a known issue with snapshots; If you restore from one after the DAO HF, you won't have the fork block header which your peers expect and they'll immediately disconnect you. This is fixed in #2098 and might need to be backported.

**Comment by rphmeier** [comment]
(duplicate of #2043)
